spring-content-autoconfigure/src/test/java/org/springframework/content/fs/boot/ContentFilesystemAutoConfigurationTest.java

```diff
@@ -90,24 +95,18 @@
 		});
 	}
 
-
+	@PropertySource("classpath:/test.properties")
```

Comment:
Interesting you got this to work.  I was unable to.

Reply:
Probably updated version of spring-boot?

Reply:
Maybe.